schedule/systemd/suse_patches-systemd_testsuite.yaml

```diff
@@ -29,6 +29,15 @@ conditional_schedule:
                 - systemd_testsuite/test_20_mainpidgames
                 - systemd_testsuite/test_21_sysusers
                 - systemd_testsuite/test_23_type_exec
+                - systemd_testsuite/test_26_setenv
+                - systemd_testsuite/test_27_stdoutfile
+                - systemd_testsuite/test_28_percentj_wantedby
+                - systemd_testsuite/test_29_udev_id_renaming
+                - systemd_testsuite/test_30_onclockchange
+                - systemd_testsuite/test_31_device_enumeration
+                - systemd_testsuite/test_32_oompolicy
+                - systemd_testsuite/test_33_clean_unit
+                - systemd_testsuite/test_34_dynamicusermigrate
```

Comment:
If this is only for Tumbleweed, the conditional name `systemd_testsuite_15SP2ORLATER` is misleading.
Let's find a better name. What do you think of `systemd_testsuite_if_VERSION`?

Reply:
I agree. Let's go for `systemd_testsuite_if_VERSION`

Reply:
Should that change be in another pull?

Reply:
> Should that change be in another pull?

No, I think it should be part of the third commit.

Reply:
Done.